tools/build_utils.py

```diff
@@ -511,7 +511,10 @@ def install_ngraph_tf(tf_version, venv_dir, ngtf_pip_whl):
 
 def download_repo(target_name, repo, version, submodule_update=False):
     # First download to a temp folder
-    call(["git", "clone", repo, target_name])
+    call([
```

Comment:
This might not work where "version" is passed as a commit# or label

Reply:
Good point. I'll close it for now.